build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+}
+
+group = 'org.example'
+version = '1.0-SNAPSHOT'
+sourceCompatibility = '14'
+targetCompatibility = '14'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.seleniumhq.selenium:selenium-java:3.141.59'
+    implementation 'com.aventstack:extentreports:3.1.5'
+    implementation 'org.apache.poi:poi:3.9'
+    implementation 'org.apache.poi:poi-ooxml:3.9'
+    implementation 'com.google.inject:guice:4.2.3'
+    implementation 'net.jmob:guice.conf:1.4.3'
+    implementation 'org.projectlombok:lombok:1.18.16'
+    annotationProcessor 'org.projectlombok:lombok:1.18.16'
+    testImplementation 'org.testng:testng:7.3.0'
+    implementation 'com.netflix.governator:governator:1.17.10'
```

Comment:
I found governator which is library for managing lifecycle of objects for Guice but unfortounatly I didn't manage to get it work with `@PreDestroy`. I will continue searching for solution later on.